restcomm/restcomm.telephony.api/src/main/java/org/restcomm/connect/telephony/api/util/B2BUAHelper.java

```diff
@@ -626,4 +632,120 @@ public static boolean isB2BUASession(SipServletMessage sipMessage) {
          return (linkedB2BUASession != null);
      }
 
+     /**
```

Comment:
this is just relocating the addHeaders logic to the B2BUAHelper to be used from anywhere as an util,right?

Reply:
Correct. We also added the specific `Route` handling to push the route